components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/APIProduct.java

```diff
@@ -63,6 +63,7 @@
 
     private JSONObject monetizationProperties = new JSONObject();
     private boolean isMonetizationEnabled = false;
+    private String versionTimestamp;
```

Comment:
we don't have versions in products, do we?

Reply:
products dont have versions, but product creation fails since api.rxt is expecting a timestamp value.

Reply:
ack.